### PR TITLE
feat(atom/progressBar): allow children to be displayed inside the com…

### PR DESCRIPTION
**Pull request by ferransimon**
…ponent

We need to allow children to be displayed instead of the percentage to cover some usecases

BREAKING CHANGE:
Now the size of the errorIcon is delagated to the client in order to simplify the code

## Atom/progressBar
**TASK**: 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description, Motivation and Context

We need this change in order to be able to display the following usecases:

image

### CODE REVIEW

**Comment by andresin87** [comment]
last manual small progress indicator looses its own %indicator text

File: components/atom/progressBar/src/CircleProgressBar/index.scss [hunk 1]
```diff
@@ -3,49 +3,71 @@
 
 .sui-AtomCircleProgressBar {
   background-color: transparent;
-  display: inline-block;
-  position: relative;
```
Comment (andresin87) [inline components/atom/progressBar/src/CircleProgressBar/index.scss:7]: this might not be removed

**Comment by andresin87** [comment]
If its a broken change the version might be modified in package. (progressBar is not already used by any other sui-components package).

The style naming might colide between versions so please, add an identifier class to prevent this posible library usage bug.

**Review by andresin87 — APPROVED** [review]

File: components/atom/progressBar/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@s-ui/react-atom-progress-bar",
-  "version": "1.10.0",
```
Comment (jcger) [inline components/atom/progressBar/package.json]: this should be set by the automatic release process, did you create a commit adding something to the breaking changes question?

**Review by jcger — APPROVED** [review]

File: components/atom/progressBar/package.json (hunk 2 above)
Comment (ferransimon) [reply components/atom/progressBar/package.json]: Yes, I did, but @andresin87 told me it wasn't automatic so now I do not know what to do xD

Comment (jcger) [reply components/atom/progressBar/package.json]: It should be working, tell me if not

Comment (andresin87) [reply components/atom/progressBar/package.json]: The automatic release process upgrades the X.[number].0 version. As long as I know, this it the requirement to create a new major

**Review by javiauso — APPROVED** [review]

File: components/atom/progressBar/package.json (hunk 2 above)
Comment (jcger) [reply components/atom/progressBar/package.json]: it changes the X if you fill the breaking changes question while running sui-studio commit command

**Review by midudev — APPROVED** [review]

**Pull request merged**